packages/jest-config/src/normalize.js

```diff
@@ -447,9 +446,9 @@ export default function normalize(options: InitialOptions, argv: Argv) {
   }
 
   const babelJest = setupBabelJest(options);
-  const newOptions: $Shape<
-    DefaultOptions & ProjectConfig & GlobalConfig,
-  > = (Object.assign({}, DEFAULT_CONFIG): any);
+  const newOptions: $Shape<DefaultOptions & ProjectConfig & GlobalConfig> = ({
+    ...DEFAULT_CONFIG,
+  }: any);
```

Comment:
do we still need to annotate this by hand?

Reply:
removed this, though 🙂